lms/djangoapps/courseware/plugins.py

```diff
@@ -287,14 +287,15 @@ def is_enabled(cls, course_key: CourseKey) -> bool:
         """
         Get open response enabled status from course overview model.
         """
-        return True
+        course = get_course_by_id(course_key)
+        return course.force_on_flexible_peer_openassessments
 
     @classmethod
     def set_enabled(cls, course_key: CourseKey, enabled: bool, user: 'User') -> bool:
         """
         Update open response enabled status in modulestore. Always enable to avoid confusion that user can disable ora.
         """
-        return True
+        raise ValueError("Flexible Peer Grading cannot be enabled/disabled via this API.")
```

Comment:
I was following the pattern that was used for proctored exams. I believe that course apps expects a `set_enabled` function so that is why it is included.